src/convertToObject.js

```diff
@@ -11,8 +11,11 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+
+const convertToObject = sourceString => Object.fromEntries(sourceString
+  .split(';')
+  .map(element => element.split(':')
+    .map(property => property.trim()))
+  .filter(item => item.length > 1));
```

Comment:
Good use of Object.fromEntries 👍 
Just watch the code style.
```suggestion
const convertToObject = sourceString => Object
  .fromEntries(sourceString
    .split(';')
    .map(element => element.split(':')
      .map(property => property.trim()))
    .filter(item => item.length > 1));
```